File: backend/src/main/java/com/music/education/Entity/ForumItemAdditional.java
package com.music.education.Entity;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class ForumItemAdditional {
    private int id;
    private int forum_item_id;

    private String file_name;

    private String file_type;

    private String file_url;

    private Boolean is_delete;
}

File: app/src/main/java/com/example/musiceducation/entity/ForumItemAdditional.kt
package com.example.musiceducation.entity

import kotlinx.serialization.Serializable

@Serializable
data class ForumItemAdditional (

    val id: Int,
    val forum_item_id: Int,
    val file_name: String,
    val file_type: String,
    val file_url: String,
    val is_delete: Boolean


)